### PR TITLE
Lo 17 create form for add products to lead

### PR DESCRIPTION
**Pull request by Diana9218**
Obtain products from DB
Products form view

### CODE REVIEW

File: resources/views/leads/add_products.blade.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+                        <div class="marco">
+                            <input id="product-name" type="checkbox" name="product-name" value="{{$product->id}}">
+                            <label for="product-name" class="info">{{$product->name}}</label>
+                            <br>
+                            <label for="product-description" class="info">{{$product->description}}</label>
+                        </div>
+                    @endforeach
+            </fieldset>
+                <br>
+                <button type="submit">Siguiente</button>
```
Comment (zahirolamadrid) [inline resources/views/leads/add_products.blade.php:29]: We use button type submit only in a form. In this case we should use a form and send de request info to the controller.

Comment (Diana9218) [reply resources/views/leads/add_products.blade.php:29]: I don't understand, in this case we should use a form?

Comment (zahirolamadrid) [reply resources/views/leads/add_products.blade.php:29]: It's a good idea to send the request to the controller and redirect to index page.

**Review by RgarcesGit — APPROVED** [review]

File: resources/views/leads/add_products.blade.php (hunk 1 above)
Comment (Diana9218) [reply resources/views/leads/add_products.blade.php:29]: ok

Comment (Diana9218) [reply resources/views/leads/add_products.blade.php:29]: can we see the view with the selected products

File: resources/views/leads/products_select.blade.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    <div>
+        <h1>{{$product_select->name}}</h1>
+    </div>
+    <div>
+        <p>{{$product_select->description}}</p>
+    </div>
+    @endforeach
+    <br>
+    <div>
+        <button><a href="#">Regresar</a></button>
```
Comment (yoyi1073) [inline resources/views/leads/products_select.blade.php]: i dont understand why does the href have a # symbol?

File: resources/views/products/index.blade.php [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
+    <style>
+        table, th, td {
+            border: 1px solid black;
+        }
+        .add-new-product{
+            width: 20%;
+            background-color: white;
+        }
+    </style>
     <div><h3>Productos</h3></div>
```
Comment (yoyi1073) [inline resources/views/products/index.blade.php]: Why does the h3 element use also a div element ?

File: resources/views/leads/products_select.blade.php (hunk 2 above)
Comment (Diana9218) [reply resources/views/leads/products_select.blade.php]: I eliminate this button
Done

File: resources/views/products/index.blade.php (hunk 3 above)
Comment (Diana9218) [reply resources/views/products/index.blade.php]: I eliminate element div

File: resources/views/leads/products_select.blade.php (hunk 2 above)
Comment (Diana9218) [reply resources/views/leads/products_select.blade.php]: Done

File: resources/views/products/index.blade.php (hunk 3 above)
Comment (Diana9218) [reply resources/views/products/index.blade.php]: done

**Review by yoyi1073 — APPROVED** [review]

File: app/Http/Controllers/AddProductsController.php [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+     * Handle the incoming request.
+     */
+    public function index(): View
+    {
+        $products = Product::all();
+
+        return view('leads.add_products', compact('products'));
+    }
+
+    public function store(Request $request): RedirectResponse|View
```
Comment (zahirolamadrid) [inline app/Http/Controllers/AddProductsController.php]: Remove redirect as type of return

File: app/Models/Leads.php [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Leads extends Model
```
Comment (zahirolamadrid) [inline app/Models/Leads.php]: In this case, we don't need a Lead model

File: app/Http/Controllers/AddProductsController.php (hunk 4 above)
Comment (Diana9218) [reply app/Http/Controllers/AddProductsController.php]: Done

File: app/Models/Leads.php (hunk 5 above)
Comment (Diana9218) [reply app/Models/Leads.php]: Done

**Review by zahirolamadrid — APPROVED** [review]

**Pull request merged**